src/gordon_gcp/plugins/janitor/__init__.py

```diff
@@ -32,18 +32,18 @@
 
 
 def get_gpubsub_publisher(config, changes_channel, **kw):
-    """Get a GPubsubPublisher client.
+    """Get a GPubsubPublisher instance.
 
     A factory function that validates configuration, creates an auth
     and pubsub API client, and returns a Google Pub/Sub Publisher
-    provider.
```

Comment:
Hmm "provider" is specific to interface implementation, but I understand the switch to 'instance'. What do you think of: 

> ...and returns a Google Pub/Sub Publisher instance, a provider of the ``IPublisherClient`` interface.

Reply:
WFM